lib/cloud/value.rego

```diff
@@ -0,0 +1,40 @@
+# METADATA
+# custom:
+#   library: true
+package lib.cloud.value
+
+import rego.v1
+
+is_unresolvable(val) if val.unresolvable
+
+# string
+
+is_empty(val) := is_equal(val, "")
+
+is_not_empty(val) := is_not_equal(val, "")
+
+# int
+
+less_than(val, other) := false if {
+	is_unresolvable(val)
+} else := val.value < other
+
+greater_than(val, other) := false if {
+	is_unresolvable(val)
+} else := val.value > other
```

Comment:
Would these cover float too?

Reply:
You mean the tests?

Reply:
Right, the good thing to do is to use epsilon, but I haven't encountered floating numbers in the checks.